src/main/resources/public/src/utils/plugins/i18n.js

```diff
@@ -0,0 +1,97 @@
+            date: 'Date',
+            organization: 'Organization | Organizations',
+            state: 'State',
+            request_access_to_dataset: 'Request for access to dataset',
+            search_organization: 'Search a organization',
+            email: 'Email',
+            model: 'No model | Model | Models',
+            search_model: 'Search a model',
+            save: 'Save',
+            search_by_organization: 'Search by organization',
+            application_name: 'Application Name',
+            search_by_application: 'Search by application',
+            organization_name: 'Organization Name',
+            siret_number: 'SIRET number',
+            data_access_check: 'Data access check',
+            authorize_access_to_all_checked_data: 'Authorize access to all checked data of "{modelName}" model',
+            field: 'Field | Fields',
+            validate: 'Validate',
+            refuse: 'Refuse',
+            processing_statistics: 'Processing Statistics',
+            since: 'Since',
+            processing_completed: 'Processing completed',
+            processed_model: 'Processed model | Processed models',
+            concerned_organization: 'Concerned organization | Concerned organizations',
+            processing_since: 'Processing since | Processings since',
+            search: 'Search',
+            by_model: 'By Model',
+            being_processed: 'being processed',
+            creation: 'Creation | Creations',
+            deletion: 'Deletion | Deletions',
+            update: 'Update | Updates',
+            by_organization: 'By Organization',
+            models_processed_for_selected_organization: 'Models processed for selected organization',
+            all: 'All',
+            for_organization: 'for Organization',
+            created_ago: 'Created ago'
+        },
+        fr: {
+            dashboard: 'Tableau de bord',
+            data_request: 'Demande de données',
+            connectors_management: 'Gestion des connecteurs',
+            statistics: 'Statistiques',
+            dc_model: 'DC Modèle',
+            date: 'Date',
+            organization: 'Organisation | Organisations',
+            state: 'Etat',
+            request_access_to_dataset: 'Demande d\'accès à un jeu de données',
+            search_organization: 'Rechercher une organisation',
+            email: 'Email',
+            model: 'Aucun modèle | Modèle | Modèles',
+            search_model: 'Rechercher un modèle',
+            save: 'Enregistrer',
+            search_by_organization: 'Recherche par organisation',
+            application_name: 'Nom d\'application',
+            search_by_application: 'Recherche par application',
+            organization_name: 'Nom d\'organisation',
+            siret_number: 'Numéro SIRET',
+            data_access_check: 'Contrôle d\'accès aux données',
+            authorize_access_to_all_checked_data: 'Autoriser l\'accès à toutes les données vérifiées du modèle "{modelName}"',
```

Comment:
I believed that "checked" meant "asked for access" (litteraly the checkbox was checked in Data Access Request). So, I'm not sure we should use "vérifiées" maybe "demandées" or "cochées" or "faisant l'objet d'une demande d'accès".